historian-server/src/main/java/com/hurence/webapiservice/timeseries/AbstractTimeSeriesExtracter.java

```diff
@@ -43,6 +49,7 @@ public void addChunk(JsonObject chunk) {
         pointCounter+=chunk.getLong(HistorianFields.RESPONSE_CHUNK_COUNT_FIELD);
         chunks.add(chunk);
         if (isBufferFull()) {
+            calculateAggreg(); // is this correct
```

Comment:
C'est pas tout a fait bon ! Ya un petit bug la car il faut aussi le faire dans flush !

Reply:
yes, i do put the calculateAggreg() in the flush also, but because in addChunk the chunks can be cleared if buffer is full so i lose the chunks so before any flush i caluculate aggregs